account/views.py

```diff
@@ -30,14 +30,21 @@
 account = Blueprint('account', __name__)
 
 
+def validate_login(form):
```

Comment:
The login function could be improved by separating the validation logic from the routing logic. This would make the code more modular and easier to test.